src/login/AzureAccountInternal.ts

```diff
@@ -0,0 +1,162 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { CancellationTokenSource, commands, ConfigurationTarget, Disposable, Event, MessageItem, QuickPickItem, window, workspace, WorkspaceConfiguration } from 'vscode';
+import { AzureAccount, AzureLoginStatus, AzureResourceFilter, AzureSession, AzureSubscription, CloudShell } from '../azure-account.api';
+import { extensionPrefix, resourceFilterSetting } from '../constants';
+import { TelemetryReporter } from '../telemetry';
+import { localize } from '../utils/localize';
+import { openUri } from '../utils/openUri';
+import { AzureLoginHelperTasks, AzureLoginStatusObj, ISubscriptionItem } from './AzureLoginHelper';
+import { addFilter, removeFilter } from './filters';
+import { getCurrentTarget } from './getCurrentTarget';
+
+export class AzureAccountInternal implements AzureAccount {
```

Comment:
Per offline discussion, use a separate legacy api class that references this class